challenge-1/theme/footer.php

```diff
@@ -15,6 +15,11 @@
 </div>
 <!-- #page -->
 
+<div class="fixed left-1/2 -translate-x-1/2 bottom-8 bg-[#222222] rounded-full px-4 py-3 flex items-center">
```

Comment:
prefer to use values defined in the tailwind conf as opposed to ad hoc definitions. In other words, `bg-[#222222]` => `bg-gray-dark` (assuming we had that color defined)